ro_help/hub/templates/ngo/detail.html

```diff
@@ -240,6 +245,11 @@ <h2 class="title">
 
         <div id="reporting" class="is-collapsible">
             <div class="card-content donation-card">
+
+                <div class="container">
+                    {% include "shared/pagination.html" with page_obj=report_items_page_obj %}
```

Comment:
It should also receive a `page_parameter_name` so we can have multiple pagination types, on the same page.